apps/docs-website/docs/setup/index.mdx

```diff
@@ -6,11 +6,7 @@ import DocCardList from "@theme/DocCardList";
 
 # 🛠️ Setup
 
-This config is opinionated and complex, so make sure to meet the [hard requirements](../requirements/hard-requirements.md) in
-your project. Then, let `@sherifforg/cli` handle the whole setup for you automatically,
-or do it yourself manually. <br />
-For greenfield projects that lack a ESLint config, the below setup options will be enough.
-However, for established codebases that already have a ESLint config setup, you should also follow the instructions
-for the [migration path](../migration-guide.mdx).
+This config is opinionated and complex, so make sure to meet the [hard requirements](../requirements/hard-requirements.md) in your project. Then, let `@sherifforg/cli` handle the setup for you automatically, or do it yourself manually.<br />
+For new projects that lack a ESLint config, the setup options listed below will be enough. However, for established codebases that already have an ESLint config setup, you should also follow the instructions for the [migration path](../migration-guide.mdx).
```

Comment:
i like the "greenfield" wording here

Reply:
I've never heard that word used before (well, outside of as a proper noun).
A quick [Google](https://www.google.com/search?q=define+greenfield) shows it's an obscure British word with a negative connotation, which doesn't seem to be the goal of that sentence. Are you trying to "exploit" new projects?

Reply:
https://naturaily.com/blog/greenfield-vs-brownfield-projects-in-it-differences-pros-cons-and-how-to-lead

Reply:
That link doesn't work, but alright.

Reply:
strange, it works for me and i even have adblocker ON.

Reply:
here is a screenshot
![ss](https://github.com/user-attachments/assets/b271d1b6-a6b8-46ec-88dd-213026ff2a27)